<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EighthTask</class>
 <widget class="QDialog" name="EighthTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Задание №8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>711</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Разработать набор функциональных объектов, выполняющих логические операции над своими операндами.
Проверить объекты на векторах типов char, int, bitset&lt;67&gt;/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delPushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить все поля</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вектор 1:
Вектор 2:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="char1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="char2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="CharOperation">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Char</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="outInt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="OutBitset">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>451</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="OutChar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вектор 1:
Вектор 2:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="int1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="int2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="intOperation">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Int</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>451</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вектор 1:
Вектор 2:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="bitset1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="bitset2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="bitsetOperation">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bitset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
